content/en/docs/concepts/scheduling/kube-scheduler.md

```diff
@@ -0,0 +1,197 @@
+according to the specific scheduling algorithm and strategy.
+
+In a cluster, Nodes that meet the scheduling requirements for a Pod
+are called _feasible_ nodes. If none of the nodes are suitable, the pod
+remains unscheduled until the scheduler is able to place it.
+
+The scheduler finds feasible Nodes for a Pod and then runs a set of
+functions to score the feasible Nodes and picks a Node with the highest
+score among the feasible ones to run the Pod. The scheduler then notifies
+the API server about this decision in a process called _binding_.
+
+Factors that need taken into account for scheduling decisions include
+individual and collective resource requirements, QoS (Quality of Service)
+requirements, hardware / software / policy constraints, affinity and
+anti-affinity specifications, data locality, inter-workload interference,
+deadlines, and so on.
+
+## Scheduling with kube-scheduler {#kube-scheduler-implementation}
+
+kube-scheduler selects a node for the pod in a 2-step operation:
+
+1. **Predicates** processing: filter the nodes
+
+2. **Priorities** processing: prioritize the filtered list of nodes
+
+
+The predicates step filters out the Nodes based on _predicates
+policies_. For example, the PodFitsResources policy checks whether a
+candidate Node has enough available resource to meet a Pod's specific
+resource requests. After this step, the node list contains any suitable
+Nodes; often, there will be more than one.
+
+In the priorities step, the scheduler ranks the remaining nodes to choose
+the most suitable Pod placement, based on some _priorities policies_. The
+scheduler will assign a score to Node that survived filtering, and
+will base this score on the active priorities policies.
+kube-scheduler assigns the Pod to the Node with the highest ranking.
+If there are more than one node with equal scores, kube-scheduler selects
+one of these at random.
+
+
+### Default policies
+
+kube-scheduler has a default set of scheduling policies.
+
+### Predicates Policies
+
+- `PodFitsHostPorts`: Checks if a Node has free ports (the network protocol kind)
+  for the Pod ports the the Pod is requesting.
+
+- `PodFitsHost`: Checks if a Pod specifies a specific Node by it hostname.
+
+- `PodFitsResources`: Checks if the Node has free resources (eg, CPU and Memory)
+  to meet the requirement of the Pod.
+
+- `PodMatchNodeSelector`: Checks if a Pod's Node {{< glossary_tooltip term_id="selector" >}}
+   matches the Node's {{< glossary_tooltip text="label(s)" term_id="label" >}}.
+
+- `NoVolumeZoneConflict`: Evaluate if the {{< glossary_tooltip text="Volumes" term_id="volume" >}}
+  that a Pod requests are available on the Node, given the failure zone restrictions for
+  that storage.
+
+- `NoDiskConflict`: Evaluates if a Pod can fit on a Node due to the volumes it requests,
+   and those that are already mounted.
+
+- `MaxAzureDiskVolumeCount`: Ensures that the number of attached Azure Disk volumes
```

Comment:
Please remove `MaxAzureDiskVolumeCount`, `MaxEBSVolumeCountPred`, and `MaxGCEPDVolumeCount`. These are deprecated in favor of `MaxCSIVolumeCount`.

Reply:
Noted